R/utils.R

```diff
@@ -56,3 +56,15 @@ can_entrace <- function(cnd) {
 }
 
 isNA <- function(x) length(x) == 1 && is.na(x)
+
+relish <- function(x, dir = testing_directory()) {
```

Comment:
Can we come up with a different name for this function

Reply:
Chutney? Condiment? Sauce?